cluster-defns/1.hopsworks-ee.yml

```diff
@@ -57,6 +57,8 @@ attrs:
     LockPagesInMainMemory: 0
     TotalMemoryConfig: 4G
     NumCPUs: 4
+  flyingduck:
+    memory_gbs: 16
```

Comment:
This is going to blow up the development VMs. If you see above we are limiting RonDB to only 4GB.

Reply:
physical memory is much lower than virtual memory. 
Till suggested it should be this high, but i can reduce it to 8GB.

Reply:
Or 4 GB